### ISSUE TITLE
Generates new buffer after closing fzf-lua

### ISSUE THREAD

**Issue by sQVe** [glepnir/dashboard-nvim#92]
👋🏼 

I'm seeing a annoying issue when running `dashboard` together with https://github.com/ibhagwan/fzf-lua. Whenever I open `fzf-lua` and open a buffer dashboard generates a new buffer. See https://github.com/ibhagwan/fzf-lua/issues/21 for further information. I do not see this issue for `fzf.vim`.

https://user-images.githubusercontent.com/2284724/126911542-23653f34-e611-487b-ae3a-7bb76fb670f1.mp4

**Comment by ibhagwan** [comment]
This is also related to [`fzf-lua` issue #20](https://github.com/ibhagwan/fzf-lua/issues/20), for some odd reason `dashboard` was preventing the floating window from being closed after wiping the buffer, I worked around it by specifically sending `nvim_win_close` to the popup window but I’m still unsure what was causing this behavior to begin with. I strongly believe the two issues are related. 

I checked my code and [`nvim-fzf`](https://github.com/vijaymarupudi/nvim-fzf) but there is nothing there to generate an additional buffer, I stopped the code in different places, some right after exiting the `fzf-lua` window and ran `ls!` and there is no mention of the new buffer, only once nvim enters `dashboard` again is when the new unnamed buffer is generated.
